hq/app/model/Serializers.scala

```diff
@@ -0,0 +1,41 @@
+package model
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Format, JsPath, Json, Reads}
+
+trait Serializers {
```

Comment:
Why is this a `trait`?

Reply:
Now this one I would like to discuss.  I appear to need correctly name implicits, in my SnykDisplay object.  I wanted to have a separate class for the serializers as they aren't likely to change and are just 'noise' from the PoV of SnykDisplay.